### PR TITLE
Allow adding new choices and editing existing choices

### PR DESCRIPTION
**Pull request by ConnorCimowsky**

### CODE REVIEW

File: src-frontend/app/styles/base.less [hunk 1]
```diff
@@ -303,44 +303,120 @@ ul.element{
...
         box-sizing: border-box;
-        border-radius: 2px;
         display: block;
-        outline: none;
-        padding: 60px 60px 60px 80px;
+        line-height: @button-size;
+        text-align: center;
+
+        position: absolute;
+        left: 0; top: 0;
```
Comment (Kapin) [inline src-frontend/app/styles/base.less]: can these be on separate lines.

File: src-frontend/app/styles/base.less [hunk 2]
```diff
@@ -303,44 +303,120 @@ ul.element{
...
+        width: @button-size; height: @button-size;
```
Comment (Kapin) [inline src-frontend/app/styles/base.less]: same as above

**Comment by Kapin** [comment]
Style nit in the LESS but otherwise LGTM

**Comment by Ommy** [comment]
I'd like if we could get an image of the new style in PRs, but LGTM

**Comment by Trinovantes** [comment]
+1

**Comment by ConnorCimowsky** [comment]
@Ommy here it is. CSS still needs to be fixed

screen shot 2015-07-06 at 4 05 34 pm

**Pull request merged**